README.md

```diff
@@ -49,6 +49,15 @@ computational effects.
 
 ### 2017
 
+* **Structured asynchrony with algebraic effects** (TyDe 2017)  
+  by Daan Leijen  
+  ([pdf](https://www.microsoft.com/en-us/research/wp-content/uploads/2017/05/asynceffects-msr-tr-2017-21.pdf))
```

Comment:
This one's a Microsoft TR, not a TyDe paper.  Could you please fix either the description or the URL?

Reply:
May I link to OpenTOC? http://www.sigplan.org/OpenTOC/tyde17.html

Reply:
Yes, but please label accordingly, e.g. `([OpenTOC redirection](…))` rather than `([pdf](…))`